lib/ansible/modules/system/user.py

```diff
@@ -553,7 +553,10 @@ def create_user_useradd(self):
 
         if self.expires is not None:
             cmd.append('-e')
-            cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+            if self.expires < time.gmtime(0):
+                cmd.append('')
```

Comment:
Wouldn't this result no value for `-e`, i.e. `useradd -e USERNAME`, which is invalid? (I haven't tested it).

Reply:
It is valid to pass an empty string to the `-e` param.

```
[root@lab-centos7 ~]# useradd testuser2 -e ''
[root@lab-centos7 ~]# ll /home
total 0
drwx------. 2 bob               users              62 Jun 27 19:00 bob
drwx------. 2 oracle            oinstall           62 Jun 26 00:15 oracle
drwx------. 2 testuser          testuser           62 Aug 15 16:07 testuser
drwx------. 2 testuser2         testuser2          62 Aug 16 20:57 testuser2
[root@lab-centos7 ~]# getent passwd testuser2
testuser2:x:1009:1011::/home/testuser2:/bin/bash
[root@lab-centos7 ~]# getent shadow testuser2
testuser2:!!:17759:0:99999:7:::
[root@lab-centos7 ~]# chage -l testuser2
Last password change					: Aug 16, 2018
Password expires					: never
Password inactive					: never
Account expires						: never
Minimum number of days between password change		: 0
Maximum number of days between password change		: 99999
Number of days of warning before password expires	: 7
```